### PR TITLE
Supports Destructing Assignment in avoid-ios9-viewport-bug

### LINKED ISSUE #3: avoid-ios9-viewport-bug dont report error using es6 destructuring assignment.
avoid-ios9-viewport-bug report error.

```
let innerWidth = window.innerWidth;
let innerHeight = window.innerHeight;
```

not report error.

```
let {innerWidth, innerHeight} = window;
```

I think, should report both pattern.

### PR DESCRIPTION
**Pull request by pirosikick**
Fix #3 

```js
let {innerWidth} = window; // error
```

### CODE REVIEW

File: src/rules/avoid-ios9-viewport-bug.js [hunk 1]
```diff
@@ -8,9 +8,24 @@ function avoidIOS9ViewportBugs(context) {
...
       let {object, property} = node;
-      if (object.name === 'window' && property.name.match(/^inner(Width|Height)$/)) {
+      if (object.name === 'window' && /^inner(Width|Height)$/.test(property.name)) {
         context.report(node, message[RegExp.$1.toLowerCase()]);
       }
+    },
+    VariableDeclarator: node => {
+      let {id, init} = node;
+
+      if (!(init.type === "Identifier" &&
```
Comment (koozaki) [inline src/rules/avoid-ios9-viewport-bug.js]: Would you like unify the quotation?
```diff
- "Identifier"
+ 'Identifier'
```

Comment (pirosikick) [inline src/rules/avoid-ios9-viewport-bug.js]: I'll fix this.

File: src/rules/avoid-ios9-viewport-bug.js [hunk 2]
```diff
@@ -1,16 +1,31 @@
...
+      let {id, init} = node;
+
+      if (!(init.type === 'Identifier' &&
+            init.name === 'window' &&
+            id.type === 'ObjectPattern')) {
+        return;
+      }
+
+      id.properties.forEach(property => {
+        if (/^inner(Width|Height)$/.test(property.key.name)) {
```
Comment (koozaki) [inline src/rules/avoid-ios9-viewport-bug.js]: bug
```javascript
let {'inner':innerWidth} = window;
```

File: src/rules/avoid-ios9-viewport-bug.js [hunk 3]
```diff
@@ -1,16 +1,31 @@
...
+    VariableDeclarator: node => {
+      let {id, init} = node;
+
+      if (!(init.type === 'Identifier' &&
+            init.name === 'window' &&
+            id.type === 'ObjectPattern')) {
+        return;
+      }
+
+      id.properties.forEach(property => {
```
Comment (koozaki) [inline src/rules/avoid-ios9-viewport-bug.js]: recommend
```javascript
if (!property.type || property.type !== 'Property' || !property.value || property.value.type !== 'Identifier') return;
```

**Pull request merged**